internal/distro/fedora/distro_test.go

```diff
@@ -408,7 +418,7 @@ func TestDistro_ManifestError(t *testing.T) {
 			}
 			testPackageSpecSets := distro_test_common.GetTestingImagePackageSpecSets("kernel", imgType)
 			_, err := imgType.Manifest(bp.Customizations, imgOpts, nil, testPackageSpecSets, nil, 0)
-			if imgTypeName == "iot-commit" || imgTypeName == "iot-container" || imgTypeName == "iot-raw-image" {
+			if imgTypeName == "iot-commit" || imgTypeName == "iot-container" || imgTypeName == "iot-raw-image" || imgTypeName == "minimal-rpm" {
```

Comment:
Do we not want to support custom mountpoints for this image type?

Reply:
Thanks for pointing this out , I am quite not sure about what customisations are to be supported by this image type. I believe that should be similar to what edge-raw-image has . @nullr0ute please share your thoughts.

Reply:
custom mount points should be fine, it should be quite flexible but with sane defaults.

Reply:
updated in the latest commit!